src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -134,14 +134,10 @@ class oopDesc;
 #define UINT64_FORMAT_W(width)   "%"   #width PRIu64
 #define UINT64_FORMAT_0          "%016"       PRIx64
 
+
```

Comment:
Extra newline added here, was that on purpose?

Reply:
No it wasn't on purpose.  I can clean this up when sending out the PR for JDK-8347990.